src/lib/viewers/media/__tests__/DashViewer-test.js

```diff
@@ -1291,6 +1299,27 @@ describe('lib/viewers/media/DashViewer', () => {
             dash.resize();
             expect(dash.mediaEl.style.width).toBe('325px');
         });
+
+        describe('Video width adjusts to account for the control bar height', () => {
+            test('Video width is affected by the control bar height if using react controls', () => {
```

Comment:
Can we paramaterize these tests?

Reply:
I actually think this is more straightforward this way. I wanted to be really explicit about the fact that the behavior is different based on whether the react controls are being used or not.